.github/workflows/ingestion-service-test.yml

```
@@ -1,40 +1,53 @@
+env:
+  DOCKERHUB_USERNAME: rudderlabs
 
 jobs:
   ingestion-service-test:
     name: ingestion-service-test
     runs-on: ubuntu-latest
     env:
       GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      TRANSFORMER_IMAGE_NAME_FOR_TEST: ${{ inputs.build_tag }}
+      TRANSFORMER_IMAGE_ALWAYS_PULL: 'false'
+      TRANSFORMER_DOCKER_USERNAME: ${{ env.DOCKERHUB_USERNAME }}
```

actionlint (1.7.4)

context "env" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "secrets", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)